recipes/approvaltests.cpp/8.0.0/test_package/test_gtest.cpp

```diff
@@ -0,0 +1,6 @@
+#define APPROVALS_GOOGLETEST
```

Comment:
According to the docs this is only needed for `Catch2`: https://github.com/approvals/ApprovalTests.cpp/blob/master/doc/GettingStarted.md#the-basics

```suggestion
```

Reply:
This line is correct - it's enabling the Googlest integration of Approvals.

Reply:
Oh! From the doc linked I thought that the `#define ...` was only needed for Catch2

Reply:
Yes, I can see how you read it that way - that's useful feedback and I'd ponder removing that possible ambiguity.

Reply:
Great! I'll mark these two comments as resolved 👍